src/search-manager/SearchManager.ts

```diff
@@ -101,7 +102,8 @@ export const SearchContextProvider: React.FC<{
   );
   // Note: SearchSortOption.RELEVANCE is special, it means "no custom sorting",
   // so we send it to useContentSearchResults as an empty array.
-  const sort: SearchSortOption[] = searchSortOrder === SearchSortOption.RELEVANCE ? [] : [searchSortOrder];
+  const sort: SearchSortOption[] = (overrideSearchSortOrder && [overrideSearchSortOrder])
+    || (searchSortOrder === SearchSortOption.RELEVANCE ? [] : [searchSortOrder]);
```

Comment:
If anyone ever uses this with `overrideSearchSortOrder == SearchSortOption.RELEVANCE`, it sends `sort=[]` which won't work.

Would prefer something like this: 

```suggestion
  const searchSortOrderToUse = overrideSearchSortOrder ?? searchSortOrder;
  const sort: SearchSortOption[] = (searchSortOrderToUse === SearchSortOption.RELEVANCE ? [] : [searchSortOrderToUse]);
```